client/post-editor/editor-title/index.jsx

```diff
@@ -52,40 +56,39 @@ export default React.createClass( {
 			const input = ReactDom.findDOMNode( this.refs.titleInput );
 			input.focus();
 		}
-	},
+	}
 
-	onChange( event ) {
+	onChange = event => {
 		if ( ! this.props.post ) {
```

Comment:
If we passed in and used `editedPostId` from the `connect` selector, we might be able to eliminate this condition.